semantic_conventions/logs/media.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
+  - id: log
+    prefix: log
+    type: attribute_group
+    brief: "Describes Log attributes"
+    attributes:
+      - id: iostream
+        requirement_level: opt_in
+        brief: >
+          The stream associated with the log. SHOULD be one of: `stdout`, `stderr`.
```

Comment:
It says SHOULD be one of the two values here. Does that mean `allow_custom_values` should be false?